static/ankersrv.css

```diff
@@ -96,9 +96,53 @@ code {
     content: url(img/icon-time-remaining.svg);
 }
 
+.icon-speedometer:before {
```

Comment:
We already use `speedometer` and `layers-half` from bootstrap icons on the "Home" tab. Why do we need an explicit svg version here? Just wondering.

Reply:
I resized the icons for the streamview and they weren't playing well once resized so I duplicated them and modified the SVG slightly so that they matched the other non-bi icons.